CODEOWNERS

```diff
@@ -73,6 +73,7 @@
 /Workbooks/Individual*Storage/** @microsoft/iaasexp
 
 /Workbooks/CosmosDb/** @microsoft/azurecosmosdb-workbooks
+/Workbooks/CosmosDb/Optimize*the*API*for*MongoDB*cost*and*performance-tsg/** @microsoft/azurecosmosdb-workbooks
```

Comment:
@gardnerjr is this the correct syntax?

Reply:
this one you don't need to do, because the line above with ** already covers everything inside workbooks/cosmosdb folder.

the one i'm talking about is further down in this file, there should be lines like

```
/gallery/workbook/microsoft.documentdb-databaseaccounts.json @microsoft/azurecosmosdb-workbooks
/gallery/cosmosdb-insights/** @microsoft/azurecosmosdb-workbooks
```
in the section that contains gallery files/folders.  that change would have cosmosdb team take ownership of the specific gallery files your team owns (and one modified in this PR)